packages/x-charts/src/ChartsSurface/ChartsSurface.tsx

```diff
@@ -38,8 +38,8 @@ const ChartsSurface = React.forwardRef<SVGSVGElement, ChartsSurfaceProps>(functi
 ) {
   const { width, height, left, right, top, bottom } = useDrawingArea();
   const { hasIntrinsicSize } = useSize();
-  const surfaceRef = useSurfaceRef();
```

Comment:
Not sure of me, but I don't see the `surfaceRef` being used elsewhere. I assumed it's dead code from the split of the container and surface

Reply:
🤔 very likely